src/modules/market/components/market-properties/market-properties.jsx

```diff
@@ -200,13 +200,22 @@ MarketProperties.propTypes = {
   updateModal: PropTypes.func.isRequired,
   linkType: PropTypes.string,
   finalizationTime: PropTypes.number,
-  buttonText: PropTypes.string,
   showAdditionalDetailsToggle: PropTypes.bool,
   showingDetails: PropTypes.bool,
-  toggleDetails: PropTypes.func,
+  toggleDetails: PropTypes.func.isRequired,
```

Comment:
`toggleDetails` is undefined when MarketProperties is used from DisputeMarketCard